src/include/duckdb/parser/tokens.hpp

```diff
@@ -8,6 +8,9 @@
 
 #pragma once
 
+// not sure why this is needed here for compiling
+//#include "duckdb/parser/statement/show_statement.hpp"
```

Comment:
Could you remove this comment?

Reply:
Sorry that I missed that. It's removed now.